### ISSUE TITLE
Qt ts plural form does not work without a placeholder

### ISSUE THREAD

**Issue by gwaldvogel** [translate/pootle#4727]
This bug was originally reported in #4690 but is moved here to have a fitting title.

It occurs when there is a plural form without a placeholder:
```xml
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de_DE" sourcelanguage="en_GB">
<context>
    <name>Attestation</name>
    <message numerus="yes">
        <source>Attachment</source>
        <translation>
            <numerusform>Anhang</numerusform>
            <numerusform>Anhänge</numerusform>
        </translation>
    </message>
</context>
</TS>
```

Pootle Version 2.7.3
TTK Version 1.13.0
```pytb
[2016-04-29T14:06:06]	system	UA	be	268349	/be/testproject/fr_BE.ts	Pièce jointe
Traceback (most recent call last):
  File "/var/www/pootle/env/bin/pootle", line 11, in <module>
    sys.exit(main())
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/runner.py", line 319, in main
    django_settings_module='pootle.settings')
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/runner.py", line 299, in run_app
    management.execute_from_command_line(command)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 533, in handle
    return self.handle_noargs(**options)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_app/management/commands/__init__.py", line 119, in handle_noargs
    self.handle_all(**options)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_app/management/commands/update_stores.py", line 67, in handle_all
    projects=self.projects)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_translationproject/models.py", line 89, in scan_translation_projects
    create_or_resurrect_translation_project(language, project)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_translationproject/models.py", line 46, in create_or_resurrect_translation_project
    tp = create_translation_project(language, project)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_translationproject/models.py", line 61, in create_translation_project
    .get_or_create(language=language, project=project)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 424, in get_or_create
    return self._create_object_from_params(lookup, params)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 457, in _create_object_from_params
    obj.save(force_insert=True, using=self.db)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_translationproject/models.py", line 302, in save
    changed = store.update_from_disk()
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_store/models.py", line 1833, in update_from_disk
    store_revision=store_revision,
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_store/models.py", line 1777, in update
    self.save(update_cache=has_changed)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_store/models.py", line 1460, in save
    super(Store, self).save(*args, **kwargs)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/base.py", line 589, in save
    force_update=force_update, update_fields=update_fields)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/base.py", line 617, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/base.py", line 679, in _save_table
    forced_update)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/base.py", line 723, in _do_update
    return filtered._update(values) > 0
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 600, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1005, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 787, in execute_sql
    cursor.execute(sql, params)
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 59, in execute
    self.db.validate_no_broken_transaction()
  File "/var/www/pootle/env/local/lib/python2.7/site-packages/django/db/backends/__init__.py", line 386, in validate_no_broken_transaction
    "An error occurred in the current transaction. You can't "
django.db.transaction.TransactionManagementError: An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.
```

**Comment by ta2-1** [comment]
@gwaldvogel thanks. Can you tell me what pre-conditions are that cause `update_stores` to be failed?

**Comment by gwaldvogel** [comment]
@ta2-1 I set up a new project in Pootle using Qt ts as data type and English as source language. Then I copy my files to the server, check that they have the correct access rights and execute update_stores.

Pootle is installed via pip inside of a virtual environment. Do you need anything else?

**Comment by ta2-1** [comment]
thanks @gwaldvogel. I hope I'll be able to reproduce.

**Comment by ta2-1** [comment]
@gwaldvogel this error can be reproduced on `MySQL` but initially I tested it on `sqlite`. 
Having to say that `update_stores` fails always for any `.ts` file: with plurals having placeholders, without plurals at all etc.
But good news is that it works on current master and it will work on 2.8.0b because we got rid of anti-pattern locking logic from `store.update()`. So there is no such transactional errors any more.
